store/tikv/pessimistic.go

```diff
@@ -95,7 +95,12 @@ func (action actionPessimisticLock) handleSingleBatch(c *twoPhaseCommitter, bo *
 			time.Sleep(300 * time.Millisecond)
 			return kv.ErrWriteConflict
 		})
+		startTime := time.Now()
 		resp, err := c.store.SendReq(bo, req, batch.region, readTimeoutShort)
```

Comment:
check err first?

Reply:
No, Should record the RPC runtime stats no matter the err result.